src/Compilers/Core/Portable/CodeGen/PrivateImplementationDetails.cs

```diff
@@ -727,7 +727,7 @@ internal MappedField(string name, Cci.INamedTypeDefinition containingType, Cci.I
         }
 
         public override ImmutableArray<byte> MappedData => _block;
-        public override bool IsReadOnly => true;
+        public override bool IsReadOnly => false;
```

Comment:
Why did this change?

Reply:
The `NullableValueType_ImplicitConversion_Byte` is a good illustration for the affected scenario. We're doing a `ldsflda` to get the field address of this compiler-generated field and getting a `Cannot change initonly field outside its .ctor.`.
But our purpose is not modify the field, merely to get a pointer to initialize a ROS. So I think this change was incorrect. I'll revert. Thanks

Reply:
I made the change back. Let me know what you think (is it better to keep the field initionly or resolve the verification error?)